.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -1,14 +1,12 @@
-Link to the milestone on Github e.g. https://github.com/nimblehq/git-templates/milestone/41?closed=1
-or
-Link to the project management tool for the release
+Link to the milestone on Github e.g. https://github.com/nimblehq/flutter-templates/milestone/41?closed=1
```

Comment:
```suggestion
Link to the milestone on Github e.g. https://github.com/nimblehq/ic-flutter-avishek/milestone/41?closed=1
```

Reply:
Updated in https://github.com/nimblehq/ic-flutter-avishek/pull/26/commits/087f4c2fa27bd775ebe3c3f2bbaaf109f6dbd957.